doubleA-N/This-is-me
# feat(navigation): implements navigation components on both desktop and mobile

## Pull request: This is me project 🚀 

### What is the problem(s)/feature(s)? 💡 

- Currently, we don't have anything on web page except the default text from Next.js. We decided to implement navigation first ti easily to manage the content on our side. This navigation will be the first UXUI implementation feature on our website

### How can we solve them? 🔧 

- Implement desktop navigation 
- Implement mobile navigation

### How can test the change? 💻 

- Checkout this branch
- Start the server
- Then, you will see the navigation components! 🎉
